payments/layer-2-payments.md

```diff
@@ -0,0 +1,62 @@
+
+The net effect is that, with significantly smaller costs and much higher transaction throughput, zkSync remains almost as secure as the Ethereum mainnet itself.
+
+### Polygon
+
+Formely dubbed "Matic", Polygon is becoming a de-facto standard for a lot of decentralized apps in the Ethereum ecosystem. Technically, it could be argued that it's not really a "Layer 2" solution as it doesn't keep in sync with the Ethereum but rather maintains its own side chain, connected to the Ethereum main chain with a set of inter-chain bridges.
+
+What makes Polygon attractive is that its chain is fully compatible with the Ethereum, making all the smart contracts, wallets and other applications currently working with Ethereum immediately usable on Polygon. One noticeable difference is that in place of the ETH token, the Polygon chain uses its own MATIC token to pay the transaction fees.
+
+The consensus on Polygon's mainnet chain is maintained by a carefully selected and somewhat decentralized set of validator nodes who use a proof-of-stake mechanism, whereby the validators stake their MATIC tokens. Because of this set-up and because no active synchronization with the Ethereum is attempted, the costs of transactions are several orders of magnitude lower than on Ethereum.
+
+A user wishing to enter and exit Polygon - that is transfer funds between Polygon and the Ethereum mainnet - needs to use a bridge - a special kind of contract that locks user's funds in one chains and issues them a respective amount in the other chain. Because those bridges need to work on the Ethereum, they're pretty costly to interact with. The good news is that a user having funds on Polygon may not have an immediate need to exit from Polygon to mainnet as there are a lot of centralized and decentralized exchanges and other kinds of DeFi solutions already operating directly _on_ the Polygon chain.
+
+## Current support in Golem
+
+The providers are ready to receive payments on any of the supported platforms - be it the Ethereum, Polygon or zkSync. Obviously, by default, they expect those payments to be made on the respective mainnet chains.
+
+On the other hand, because we assume requestors will first start by testing the ground using their just-created apps on a testnet, the default network for those nodes is zkSync/Rinkeby.
+
+When you run `yagna payment fund` \(you don't have to add `--driver=zksync` since it's currently the default\) on testnet, Golem initializes a new account from our custom faucet \(a service that transfers test tokens to an address that asks for it\) which provides it with test GLM tokens that are already transferred to zkSync. Then all payments are performed through zkSync and in consequence, all the providers receive all the payments to their zkSync accounts.
```

Comment:
I'm not sure we should explicitly state the zkSync being "current default" here, as we will need to remember to keep it in sync with evolving Golem (where the defaults may change). I would just state, that the selectionof payment platform is done using these parameters, and that there is always some default platform (probably one of the supported testnets) which will be used when no specific value is indicated.

What do you think?

Reply:
agreed